src/Synetic/JsonApiClient/ResponseParser.php

```diff
@@ -16,18 +16,18 @@ class ResponseParser implements ResponseParserInterface {
    * {@inheritdoc}
    */
   public function responseHasErrors(Response $response) {
-    if ($response->getStatusCode() !== 200) {
-      return true;
+    if ($response->getStatusCode() < 200 || $response->getStatusCode() > 299) {
+      return TRUE;
```

Comment:
This is not drupal code, so all booleans can stay lowercased.

Reply:
Modified in next commit